app/serializers/assignment_repo_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AssignmentRepoSerializer < ActiveModel::Serializer
+  attributes :id
+  attributes :username
+  attributes :repoUrl
+  attributes :displayName
+
+  def username
+    object.user.github_user.login
+  end
+
+  # rubocop:disable MethodName
+  def repoUrl
+    object.github_repository.html_url
+  end
+
+  def displayName
+    object.user.github_user.name || ""
+  end
+  # rubocop:enable MethodName
```

Comment:
3 API requests per repo 😱 

There's not much that can be done though, unless there's any key here we can remove.

Reply:
We can probably return fewer things but I'll update that in a future PR after we finalized Classroom-Desktop!